### DISCUSSION TITLE
How to Add Nutrition Information

### DISCUSSION THREAD

**Discussion by Axehaft** [TandoorRecipes/recipes#2667] [Q&A]
Is there any way to add nutrition information?  I feel like there should be, but I'm just not seeing how.  Even if I could add a photo of the nutrition, that would be useful.

**Comment by smilerz** [comment]
It's under additional options.

**Reply by Axehaft** [reply]
This is my 'Additional Properties'.  I just don't see how to use this for nutrition?
<img width="648" alt="image" src="https://github.com/TandoorRecipes/recipes/assets/46401365/5198a764-3dfc-45cf-b58a-3bda5bf8f602">

**Reply by smilerz** [reply]
I apologize, it's above, not in additional options. 

![image](https://github.com/TandoorRecipes/recipes/assets/1808639/7e3e7cdc-11ec-473c-af3a-8fbfb99b5717)

There should be some properties already, such as calories,, carbs, etc.  If not, you can add additional ones.

**Reply by Axehaft** [reply]
Thanks ... mine does not have anything there.  Is there instruction somewhere about how to use the Properties function?
![image](https://github.com/TandoorRecipes/recipes/assets/46401365/a0f687a5-e27a-465b-80d9-14d12d85b2f3)

**Reply by smilerz** [reply]
Navigate to /list/property-type/ and add new properties there (note 'order' is a required field even though it isn't marked)

**Reply by Axehaft** [reply]
Ahh.  Excellent.  I see it now.  Is there a way to add properties to ingredients and have it calculate that way?  Or just on the whole recepie?

**Reply by smilerz** [reply]
you can add properties to food - not sure if they roll up to the recipe yet or not.

Edit food by selecting food from within the object list view (drop down next to Property Type) or the briefcase at the top right.
